### ISSUE TITLE
Protocol Buffer support

### ISSUE THREAD

**Issue by lassik** [Unibeautify/beautifier-clang-format#61]
clang-format can also format Protocol Buffer definition files but it's not currently included in the languages list in `src/index.ts`.

**Comment by szeck87** [comment]
https://github.com/Unibeautify/unibeautify/blob/master/src/languages.json#L6120-L6128 should probably be updated.  Extensions at the very least.

**Comment by Glavin001** [comment]
Pull Requests very welcome!

- [ ] Update language:
  - https://github.com/Unibeautify/unibeautify/blob/master/src/languages.json#L6120-L6128
  - [ ] File extensions (has 1 currently, any more @lassik ?)
  - [ ] VSCode language IDs
- [ ] Add `Protocol Buffer` language support:
  - https://github.com/Unibeautify/beautifier-clang-format/blob/master/src/index.ts#L48-L54
- [ ] Add test(s)

**Comment by lassik** [comment]
Can this be done automatically by [scripts/update-languages.ts](https://github.com/Unibeautify/unibeautify/blob/master/scripts/update-languages.ts)? Protocol Buffer definitions are now recognized by GItHub Linguist.

**Comment by Glavin001** [comment]
Yes, good point. Running the script would be the ideal process. 🎉
